CHANGELOG.md

```diff
@@ -1,13 +1,18 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
+## v2.9.0
+
+### Map
+
+* `NavigationMapView.removeAlternativeRoutes()` and `NavigationMapView.removeContinuousAlternativeRoutesDurations()` were made public to provide a way to remove previously shown alternative routes and alternative routes duration annotations, respectively. ([#4134](https://github.com/mapbox/mapbox-navigation-ios/pull/4134))
+
```

Comment:
Looks like this should not be a part of `v2.8.0`.

Reply:
Thanks for the suggestion, this part has been updated and moved to v2.8.0